usr/share/rear/conf/default.conf

```diff
@@ -373,8 +373,11 @@ LANG_RECOVER=C
 FDRUPSTREAM_INSTALL_PATH="/opt/fdrupstream"
 COPY_AS_IS_FDRUPSTREAM=( "$FDRUPSTREAM_INSTALL_PATH" )
 COPY_AS_IS_EXCLUDE_FDRUPSTREAM=( "$FDRUPSTREAM_INSTALL_PATH/usserver" )
-#CHECK_CONFIG_FILES_FDRUPSTREAM=( "$FDRUPSTREAM_INSTALL_PATH/upstream.cfg" "$FDRUPSTREAM_INSTALL_PATH/uscmd1" "$FDRUPSTREAM_INSTALL_PATH/usdaemon2" )
+CHECK_CONFIG_FILES_FDRUPSTREAM=( "$FDRUPSTREAM_INSTALL_PATH/uscmd*" "$FDRUPSTREAM_INSTALL_PATH/usdaemon*" "$FDRUPSTREAM_INSTALL_PATH/us" "$FDRUPSTREAM_INSTALL_PATH/us1" "$FDRUPSTREAM_INSTALL_PATH/us.ser" "$FDRUPSTREAM_INSTALL_PATH/*.so" "$FDRUPSTREAM_INSTALL_PATH/usudb*" "$FDRUPSTREAM_INSTALL_PATH/*.dat" "$FDRUPSTREAM_INSTALL_PATH/*.cfg" )
```

Comment:
Another thought: Reading `layout/compare/default/51_compare_files.sh` makes me doubt if your quoted patterns actually work. Did you actually test that (modify one of those files matching the patterns)? Maybe better not quote those pattern strings and let Bash do its thing.